examples/go/vm-reconfig-via-tag/handler/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/codegold79/vcenter-event-broker-appliance/examples/go/vm-reconfig-via-tag/handler
+
+go 1.14
+
+require (
+	github.com/openfaas-incubator/go-function-sdk v0.0.0-20200405082418-b31e65bf8a33
```

Comment:
When I run `go mod tidy` it looks slightly different

```bash
diff --git a/examples/go/vm-reconfig-via-tag/handler/go.mod b/examples/go/vm-reconfig-via-tag/handler/go.mod
index 52c02c7..540bfe0 100644
--- a/examples/go/vm-reconfig-via-tag/handler/go.mod
+++ b/examples/go/vm-reconfig-via-tag/handler/go.mod
@@ -3,9 +3,7 @@ module github.com/codegold79/vcenter-event-broker-appliance/examples/go/vm-recon
 go 1.14
 
 require (
-	github.com/openfaas-incubator/go-function-sdk v0.0.0-20200405082418-b31e65bf8a33
 	github.com/openfaas/templates-sdk v0.0.0-20200723110415-a699ec277c12
 	github.com/pelletier/go-toml v1.8.0
 	github.com/vmware/govmomi v0.23.1
-	go.opencensus.io v0.22.4
 )
```

(same for `go.sum`)

Reply:
I'll run go mod tidy and add changes to commit.